synapseformation/create.py

```diff
@@ -271,7 +296,7 @@ def get_or_create_challenge(self, **kwargs) -> dict:
             challenge = self._create_challenge(**kwargs)
         except SynapseHTTPError as err:
             # Must check for 409 error
-            if err.response.status_code != 409:
+            if err.response.status_code != 400:
                 raise err
             if self.only_create:
                 raise ValueError(f"{str(err)}. To use existing entities, "
```

Comment:
FYI, f-strings automatically call `__str__()` on an object before interpolating the string (demonstrated below). So, you don't need to include the `str()` function call in the future. As an aside, if you happen to need `__repr__()` instead of `__str__()`, you can use the `!r` notation, shown below. 

```
>>> e = ValueError("This is a test.")

>>> f"before {str(e)} after"
'before This is a test. after'

>>> f"before {repr(e)} after"
"before ValueError('This is a test.') after"

>>> f"before {e} after"
'before This is a test. after'

>>> f"before {e!r} after"
"before ValueError('This is a test.') after"
```

Reply:
This is cool! i did not know this!